README.md

```diff
@@ -43,8 +43,13 @@ Optionally (but recommended), you can preinstall the following:
 * ansible>=2.7.0
 * python-openstackclient
 * python-requests
-* python2-jmespath
-* openstacksdk
+* python-jmespath
+* python-openstacksdk
+* python-netaddr
```

Comment:
I thougth this was brougth by ansible already as dependency?

Reply:
Nope, these are necessary by ansible, but ansible doesn't get them automatically.

Reply:
Can we instead load the requirements.txt file? It would be easier, and more sustainable. :)

Reply:
Doesn't seem so. At least on Tumbleweed I needed to manually install it.

Reply:
> Can we instead load the requirements.txt file? It would be easier, and more sustainable. :)

Which requirements file? If they are really strictly required by ansible we should fix the ansible packages. I thought they were just optional dependencis for ansible.

Reply:
In scripts library we have a requirements.txt file to bootstrap the env. It has this content in.
It's just that README.md can get out of sync.

I think it's worth using a real docs system, but in the meantime I propose we create a card for it.

Reply:
Card created.